pkg/controller/versions/strategy_v2_6.go

```diff
@@ -461,6 +462,18 @@ func (v *versionStrategyV2_6) Render(ctx context.Context, cr *common.ControllerR
 		return nil, fmt.Errorf("could not set field meshConfig.ingressControllerMode: %v", err)
 	}
 
+	// Enable FIPS automatically if necessary
+	data, err := ioutil.ReadFile("/proc/sys/crypto/fips_enabled")
+	if err != nil {
+		return nil, fmt.Errorf("could not read file /proc/sys/crypto/fips_enabled: %v", err)
```

Comment:
Do all kernels expose this file?

Reply:
This and the other errors could also be converted to just a warning. If that file doesn't exist, it should not block other things to happen.

Reply:
Thanks! It wouldn't work on regular clusters. I logged a message on info level, because our logger does not support warnings.